tests/Frontend/PathSpec.hs

```diff
@@ -33,6 +35,19 @@ import Network.Wai
 import Test.Hspec.Wai (get, post, shouldRespondWith)
 import qualified Test.Hspec.Wai.QuickCheck as Wai (property)
 
+isBrokenPath :: Main -> Bool
```

Comment:
Can you add some explanation why the random values here broke the test cases? Which assumption is not hold?